### PR TITLE
Fixes Issue #538

### PR DESCRIPTION
**Pull request by robertojrojas**
Fixes Issue #538

### CODE REVIEW

File: check/check.go [hunk 1]
```diff
@@ -180,7 +179,9 @@ func (c *Check) run() State {
 		}
 	}
 
-	if finalOutput == nil {
+	if finalOutput != nil {
+		glog.V(3).Infof("Check.ID: %s Command: %q TestResult: %t Score: %q \n", c.ID, lastCommand, finalOutput.testResult, c.State)
```
Comment (danielpacak) [inline check/check.go]: I see that you want to print score but you pass `c.State` as arg?

File: check/check.go [hunk 2]
```diff
@@ -242,7 +243,7 @@ func isShellCommand(s string) bool {
 
 	out, err := cmd.Output()
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "%s\n", err)
+		fmt.Fprintf(os.Stderr, "failed to check if command: %q is valid %v\n", s, err)
 		os.Exit(1)
```
Comment (danielpacak) [inline check/check.go]: NIT: To facilitate testing I'd return a wrapped error rather than call os.Exit here.

File: cmd/root.go [hunk 3]
```diff
@@ -64,7 +64,7 @@ var RootCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		benchmarkVersion, err := getBenchmarkVersion(kubeVersion, benchmarkVersion, viper.GetViper())
 		if err != nil {
-			exitWithError(err)
+			exitWithError(fmt.Errorf("unable to get benchmark version. error: %v", err))
```
Comment (danielpacak) [inline cmd/root.go]: NIT: I don't like that we os.Exit in such common code / package. At some point it should be refactored to return wrapped errors so you could properly test it. I know it's not easy though.

For now you could have `exitWithErrorf("unable to get benchmark version: %v", err)`

File: check/check.go (hunk 1 above)
Comment (robertojrojas) [reply check/check.go]: Yes, that's a typo. Good catch!

File: cmd/root.go (hunk 3 above)
Comment (robertojrojas) [reply cmd/root.go]: Yeah, we're using that function a lot in **kube-bench**. A bit of refactoring would need to be done. For this PR, I just wanted to add a bit more information to help user troubleshooting.

**Pull request merged**